config/locales/nl.yml

```
@@ -38,3 +38,5 @@ nl:
     query_saved_message: Query opgeslagen
     select_all: Selecteer alle rijen
     deselect_all: Deselecteer alle rijen
+    expand: Uitbreiden
+    collapse: Ineenstorting
```

Not sure about the other languages, but in Dutch this would be a better translation:
```suggestion
    collapse: Inklappen
```

---

thank you, I have fixed